hwbench/environment/cpu_info.py

```diff
@@ -60,7 +60,10 @@ def get_model(self) -> int:
         return int(self._mandatory_spec("Model"))
 
     def get_model_name(self) -> str:
-        return self._mandatory_spec("Model name")
+        try:
+            return self._mandatory_spec("Model name")
+        except ValueError as _:
+            return self._mandatory_spec("BIOS Model name")
```

Comment:
Are you 100% sure this attribute is always present? On my machine in WSL, I have the `Model name` but not the `BIOS Model name`

Reply:
This is a fallback, and we added it just in case the "Model name" does not exist. I don't expect it to be used very often, but since "Model name" is treated by hwbench as a mandatory key of the `lscpu` output, it provided a fallback to have hwbench run on this machine.